app/controllers/application_controller.rb

```diff
@@ -1,33 +1,23 @@
 class ApplicationController < ActionController::Base
   CLAIM_TIMEOUT_LENGTH_IN_MINUTES = 30
   CLAIM_TIMEOUT_WARNING_LENGTH_IN_MINUTES = 2
-  ADMIN_TIMEOUT_LENGTH_IN_MINUTES = 30
 
   http_basic_authenticate_with(
     name: ENV["BASIC_AUTH_USERNAME"],
     password: ENV["BASIC_AUTH_PASSWORD"],
     if: -> { ENV["BASIC_AUTH_USERNAME"].present? },
   )
 
-  helper_method :current_policy_routing_name, :admin_signed_in?, :admin_timeout_in_minutes, :claim_timeout_in_minutes, :timeout_warning_in_minutes
-  before_action :end_expired_admin_sessions
+  helper_method :current_policy_routing_name, :claim_timeout_in_minutes, :timeout_warning_in_minutes
   before_action :end_expired_claim_sessions
-  before_action :update_last_seen_at
+  after_action :update_last_seen_at
 
   private
 
   def current_policy_routing_name
     params[:policy]
   end
 
-  def admin_signed_in?
-    session.key?(:user_id)
-  end
-
-  def admin_timeout_in_minutes
-    ADMIN_TIMEOUT_LENGTH_IN_MINUTES
-  end
-
   def claim_timeout_in_minutes
```

Comment:
I'm wondering if it still makes sense for the claim-related timeout code to remain in `ApplicationController`? It would seem more symmetrical for the claim journey controllers to have their own namespace and base class, the same way that the admin controllers do. This means that the admin controller won't have access to a load of methods that it never uses, making it easier to reason about.

Thinking out loud but there seems to be a bit of an asymmetry now; it seems like pages in the claim journey are treated as the default, and that admin pages are then a specialisation of this

Reply:
Interesting suggestion. I've created a commit that extracts a separate base controller for the public-facing controllers: https://github.com/DFE-Digital/dfe-teachers-payment-service/commit/919d546d233c974ed1f5e04ed1b75bd414476020

I'm on the fence about which I prefer. In some ways, it's better because, as you say, admin controllers no longer have code in them that are not relevant to the admin interface. On the downside we've introduced more indirection and additional inheritance. what do you think?